NAMESPACE

```diff
@@ -1,5 +1,6 @@
 # Generated by roxygen2: do not edit by hand
 
+export(app_version)
```

Comment:
Why do you need to export the function? Starting the app was convenient by typing `ESQapp::` in the console, and then it automatically suggested the only available function `run_app`. With this new exported function, the first suggestion is always `app_version` and the user must make additional movements to run the app :)